packages/app-store/routing-forms/pages/reporting/[...appPages].tsx

```diff
@@ -56,7 +57,11 @@ const Result = ({ formId, jsonLogicQuery }: { formId: string; jsonLogicQuery: Js
   const headers = useRef<string[] | null>(null);
 
   if (!isLoading && !data) {
-    return <div>Error loading report {error?.message} </div>;
+    return (
+      <LicenseRequired>
```

Comment:
Do we need this as well ? There is already a LicenseRequired on the parent component

Reply:
Removed!